packages/twenty-emails/src/emails/validate-trust-domain.email.tsx

```diff
@@ -0,0 +1,67 @@
+import { t } from '@lingui/core/macro';
+import { Trans } from '@lingui/react/macro';
+import { Img } from '@react-email/components';
+import { emailTheme } from 'src/common-style';
+
+import { BaseEmail } from 'src/components/BaseEmail';
+import { CallToAction } from 'src/components/CallToAction';
+import { HighlightedContainer } from 'src/components/HighlightedContainer';
+import { HighlightedText } from 'src/components/HighlightedText';
+import { Link } from 'src/components/Link';
+import { MainText } from 'src/components/MainText';
+import { Title } from 'src/components/Title';
+import { WhatIsTwenty } from 'src/components/WhatIsTwenty';
+import { capitalize } from 'src/utils/capitalize';
+import { APP_LOCALES, getImageAbsoluteURI } from 'twenty-shared';
+
+type SendTrustDomainValidationProps = {
```

Comment:
Still need to rename Trustxxx before we can merge?

Reply:
Yes it's in the next PR